python/pyarrow/_parquet.pyx

```diff
@@ -96,72 +98,87 @@ cdef class Statistics(_Weakrefable):
 
     @property
     def has_min_max(self):
+        """bool: whether min and max are present"""
```

Comment:
Huh, did not realize numpydoc recommended this style for property docstrings. It looks like the sentences should still end in a period, though?

Reply:
Yeah I don't think it's adopted anywhere else in Arrow, but it seemed like the best way to do it to me. Using a Return section felt verbose and IIRC doesn't show the type in the attribute summary table.

Reply:
Maybe doing something like `Whether min and max are present (bool).` is also an option? 

(I wanted to add the same comment about this style not being consistent with numpydoc)

Reply:
I actually mean that as-is, this seems to be the recommended numpydoc style, at least as parsed by Napoleon: https://github.com/sphinx-contrib/napoleon/blob/dce30797b7a229ccebda4030f65482d501427794/docs/source/example_numpy.py#L280-L282

It just looks quite different from the rest of numpydoc style.

Reply:
What Joris suggested is also used by some projects: https://developer.lsst.io/python/numpydoc.html#documenting-class-properties

I'm good with either way. Do we prefer putting the type at the end?

Reply:
Oh huh.

I think what Joris suggests looks a bit better, but does numpydoc/napoleon parse it properly? If not I'd rather stick with how it is.

Reply:
> does numpydoc/napoleon parse it properly?

Either version isn't really parsed into anything special. Hence why I have to have the `` :class:`XXX` `` to get cross references.

Reply:
In that case I slightly prefer what Joris has but how it is is also fine (so long as it's punctuated)